logging/google/cloud/logging/_gax.py

```diff
@@ -211,13 +212,22 @@ def sink_create(self, project, sink_name, filter_, destination):
         :type destination: str
         :param destination: destination URI for the entries exported by
                             the sink.
+
+        :type unique_writer_identity: bool
+        :param unique_writer_identity: (Optional) determines the kind of
+                                       IAM identity returned as
+                                       writer_identity in the new sink.
         """
         options = None
         parent = 'projects/%s' % (project,)
         sink_pb = LogSink(name=sink_name, filter=filter_,
                           destination=destination)
+        uwi = unique_writer_identity
         try:
-            self._gax_api.create_sink(parent, sink_pb, options=options)
+            self._gax_api.create_sink(parent,
+                                      sink_pb,
+                                      unique_writer_identity=uwi,
+                                      options=options)
```

Comment:
If line length issues are the cause of your `uwi` variable, I suggest instead:

```python
            self._gax_api.create_sink(
                parent,
                sink_pb,
                unique_writer_identity=unique_writer_identity,
                options=options,
            )
```

(This way of indenting is probably the "best" in the sense that a machine could easily follow this strategy, though PEP8 accepts many different ways to do line continuation.)